deployment/local-k8s/README.md

```diff
@@ -111,16 +117,24 @@ These, in turn, will call the scripts in [k8s bash-scripts](../../example-runner
    <or>
    bash deployment/local-k8s/example-runner/runK8sExample.sh
    ```
+   If the example is deployed into a specific namespace then add the value to the end of the command:
+   ```bash
+   bash deployment/local-k8s/example-runner/runFormattedK8sExample.sh namespace
+   <or>
+   bash deployment/local-k8s/example-runner/runK8sExample.sh namespace
+   ```
    
 1. If you have run the Formatted example, and want to verify that everything has run as expected, Palisade has a validation script:
     ```bash
    bash deployment/local-k8s/example-runner/verify.sh
```

Comment:
The verify.sh can fail if the user has been using a namespace up to this point.  I think that we need to consider both changing the verify.sh file and this section of the readme.

Reply:
Added relevant details to the ReadMe file. The verify script works as expected.